src/main/java/uk/gov/hmcts/divorce/bulkscan/transformation/PaperFormDetailsTransformer.java

```diff
@@ -0,0 +1,160 @@
+        );
+
+        applicant1StatementOfTruth(ocrDataFields, caseData);
+        applicant2StatementOfTruth(ocrDataFields, caseData);
+        setCourtFee(ocrDataFields, caseData);
+        return transformationDetails;
+    }
+
+    private Set<FinancialOrderFor> applicantSummaryFinancialOrderFor(String applicantPrayerFinancialOrderFor) {
+        Set<FinancialOrderFor> financialOrdersFor = new HashSet<>();
+        if (THE_SOLE_APPLICANT_OR_APPLICANT_1.equalsIgnoreCase(applicantPrayerFinancialOrderFor)) {
+            financialOrdersFor.add(APPLICANT);
+        } else if (FOR_THE_CHILDREN.equalsIgnoreCase(applicantPrayerFinancialOrderFor)) {
+            financialOrdersFor.add(APPLICANT);
+            financialOrdersFor.add(FinancialOrderFor.CHILDREN);
+        } else if (THE_SOLE_APPLICANT_OR_APPLICANT_1_FOR_THE_CHILDREN.equalsIgnoreCase(applicantPrayerFinancialOrderFor)) {
+            financialOrdersFor.add(FinancialOrderFor.CHILDREN);
+        }
+        return financialOrdersFor;
+    }
+
+    private void applicant1StatementOfTruth(OcrDataFields ocrDataFields, CaseData caseData) {
+        if (!toBoolean(ocrDataFields.getSoleApplicantOrApplicant1StatementOfTruth())
+            && !toBoolean(ocrDataFields.getSoleApplicantOrApplicant1LegalRepStatementOfTruth())) {
+            caseData.getTransformationAndOcrWarnings().add("Please review statement of truth for applicant1 in scanned form");
+        }
+        if (!toBoolean(ocrDataFields.getSoleApplicantOrApplicant1Signing())
+            && !toBoolean(ocrDataFields.getSoleApplicantOrApplicant1OrLegalRepSignature())) {
+            caseData.getTransformationAndOcrWarnings().add("Please review statement of truth signing for applicant1 in scanned form");
+        }
+        if (isEmpty(ocrDataFields.getSoleApplicantOrApplicant1OrLegalRepFullName())) {
+            caseData.getTransformationAndOcrWarnings().add("Please review sole or applicant1/legal representative name in scanned form");
+        }
+
+        caseData.getPaperFormDetails().setApplicant1SigningSOT(
+            from(toBoolean(ocrDataFields.getSoleApplicantOrApplicant1Signing()))
+        );
+        caseData.getPaperFormDetails().setApplicant1LegalRepSigningSOT(
+            from(toBoolean(ocrDataFields.getSoleApplicantOrApplicant1OrLegalRepSignature()))
+        );
+        caseData.getPaperFormDetails().setApplicant1LegalRepPosition(ocrDataFields.getSoleApplicantOrApplicant1LegalRepPosition());
+
+        String day = ocrDataFields.getStatementOfTruthDateDay();
+        String month = ocrDataFields.getStatementOfTruthDateMonth();
+        String year = ocrDataFields.getStatementOfTruthDateYear();
+        if (isEmpty(day) || isEmpty(month) || isEmpty(year)) {
+            caseData.getTransformationAndOcrWarnings().add("Please review statement of truth date for applicant1 in scanned form");
+        }
+        caseData.getPaperFormDetails().setApplicant1SOTSignedOn(
```

Comment:
This is getting set here and then set again in applicant2StatementOfTruth. Not sure if that is by design